projects/angular/src/data/datagrid/datagrid-row.spec.ts

```diff
@@ -543,6 +543,123 @@ export default function (): void {
         context.detectChanges();
       }
     });
+
+    describe('Details', function () {
+      let context: TestContext<ClrDatagridRow<Item>, DatagridWithDisabledOrHiddenDetails>;
+
+      beforeEach(function () {
+        context = this.create(ClrDatagridRow, DatagridWithDisabledOrHiddenDetails, DATAGRID_SPEC_PROVIDERS);
+        context.detectChanges();
+      });
+
+      it('displays a clickable double caret when the row have details', function () {
+        expect(context.clarityElement.querySelector('button cds-icon[shape^=angle-double]')).not.toBeNull();
```

Comment:
Can we just check for the actual button?
```suggestion
        expect(context.clarityElement.querySelector('button.datagrid-detail-caret-button')).not.toBeNull();
```

Reply:
Corrected in commit 00e868d